src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractFileSetCheckTest.java

```diff
@@ -250,7 +249,7 @@ protected void processFiltered(File file, FileText fileText) {
     public static class MultiFileViolationFileSetCheck extends AbstractFileSetCheck {
 
         private static final String MSG_KEY = "Violation.";
-        private static int finishProcessingCount;
+        private int finishProcessingCount;
```

Comment:
I have no idea why this field was `static`. It has only one usage, so it worked fine until someone (Pitest actually) invoked the test twice.

Reply:
Please move this to a separate commit.

Reply:
done